xapian-applications/omega/omegatest

```diff
@@ -1451,6 +1451,36 @@ printf '$hitlist{$list{$terms{T},|}/$field{tag1}|$field{tag2}|$field{tag3}}' > "
 testcase 'Tone/one|two|three' B=Tone
 testcase 'Tthree|Ttwo/one|two|three' B=Ttwo B=Tthree
 
+
+
+printf 'text: index\n boost: boolean\npfx_boost: boolean=T' > "$TEST_INDEXSCRIPT"
+rm -rf "$TEST_DB"
+$SCRIPTINDEX "$TEST_DB" "$TEST_INDEXSCRIPT" > /dev/null <<'END'
+text=one
+text=two
+boost=three
+
+text=one
+text=nine
+boost=four
+boost=five
+boost=six
+pfx_boost=ten
+
+END
+
+# Test CGI param AUTOBOOST boost boolean terms.
+# Currently this tests don't work for following weighting schemes
+# dicecoeff dph ineb2 lm inl2 pl2plus pl2 tradweight
+for i in coord tfidf bm25 bm25plus
```

Comment:
I got the implementation for few initial weighting schemes. I can edit try doing similar for other weightings schemes if this is a correct approach.